### PR TITLE
Fix hover styling for CategoriesWidget items

### PR DESCRIPTION
**Pull request by Geecek**

### CODE REVIEW

File: assets/scss/main.scss [hunk 1]
```diff
@@ -170,6 +157,29 @@ a {
...
+}
+
+.link-no-decoration {
+  color: inherit;
+  text-decoration: none;
+
+  &:hover, &:active, &:visited {
+    text-decoration: none;
+  }
+}
```
Comment (k911) [inline assets/scss/main.scss]: Just to note: when you just move logic, it is harder to notice any changes in code, so please do not do that unless necessary

Comment (FieryCod) [reply assets/scss/main.scss]: It was me how told him to do that. Sorry @k911

**Comment by FieryCod** [comment]
First of all welcome in a team. We really appreciate your contribution. Wish you much success and more contributions. Glad we got you.


PS: @Geecek Please always add all team to reviewers.

File: assets/scss/main.scss [hunk 2]
```diff
@@ -170,6 +157,29 @@ a {
...
+
+.hover-list-item {
+  &:hover,
+    &:focus {
+      color: $gray-50;
+      background-color: $gray-10;
+      text-decoration: underline;
```
Comment (FieryCod) [inline assets/scss/main.scss]: Please remove underline. It looks really bad with that property, adding gray color as you did is sufficient.

File: assets/scss/main.scss [hunk 3]
```diff
@@ -170,6 +157,29 @@ a {
...
+.link-no-decoration {
+  color: inherit;
+  text-decoration: none;
+
+  &:hover, &:active, &:visited {
+    text-decoration: none;
+  }
+}
+
+.hover-list-item {
```
Comment (FieryCod) [inline assets/scss/main.scss]: Please name it `widget-item` delete &:focus.
Also, you can use that class in job offers components. Do it, please.

File: components/widgets/partials/JobOffersWidgetItem.vue [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="job-offers-widget-item">
+  <div class="job-offers-widget-item widget-item">
```
Comment (FieryCod) [inline components/widgets/partials/JobOffersWidgetItem.vue]: You should delete the hover property in scss down here. Line 101.

Comment (Geecek) [reply components/widgets/partials/JobOffersWidgetItem.vue]: Yes, good point. There was another unnecessary &:hover in ProjectsWidgetItem. Removed them both.

Comment (FieryCod) [reply components/widgets/partials/JobOffersWidgetItem.vue]: Nice good job!

**Comment by FieryCod** [comment]
@Geecek now we have to wait for others to approve :)

**Pull request merged**